docs/operators/s3outputmodule.md

```diff
@@ -0,0 +1,149 @@
+S3OutputModule
+==============
+
+### About Amazon S3
+-------------------
+
+**Amazon S3 (Simple Storage Service)** is a web service interface that you can use to store and retrieve any amount of data at any time from anywhere on the web offered by Amazon Web Services.
+
+### About S3 Output Module
+--------------------------
+
+Purpose of S3Output module is to upload files/directories into Amazon S3 bucket using multi part upload feature.
+
+S3Output module is **fault-tolerant**, **statically/dynamically partitionable** and **exactly once upload**.
+
+Module class is **S3OutputModule** located in the package **org.apache.apex.malhar.lib.fs.s3**.
+
+Please refer to [github URL](https://github.com/apache/apex-malhar/blob/master/library/src/main/java/org/apache/apex/malhar/lib/fs/s3/S3OutputModule.java) for **S3OutputModule**.
```

Comment:
Suggest combining this with the previous sentence, e.g.:
Module class is .... ; please refer to [github URL](https://......).